datetime/src/test/java/com/fasterxml/jackson/datatype/jsr310/ModuleTestBase.java

```diff
@@ -28,7 +29,8 @@ protected static ObjectMapper newMapper() {
 
     protected static MapperBuilder<?,?> newMapperBuilder() {
         return JsonMapper.builder()
-                .addModule(new JavaTimeModule());
+                .addModule(new JavaTimeModule())
+                .defaultLocale(Locale.ENGLISH);
```

Comment:
the tests (parsing dates like `1 Jan 2020`) were failing when you ran them on a pc with a locale different than English